aspnetcore/grpc/interprocess/Server.csproj

```diff
@@ -0,0 +1,9 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <ItemGroup>
+    <FrameworkReference Include="Microsoft.AspNetCore.App" />
+    <PackageReference Include="Grpc.AspNetCore" Version="2.49.0" />
```

Comment:
@JamesNK, for Grpc.AspNetCore isn't the latest version available 2.47.0?  Maybe a typo?  Looking at NuGet plus including prerelease I still only see 2.47.0 as the latest.